earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp

```diff
@@ -187,7 +200,14 @@ AssetVersionImplD::SetState(AssetDefs::State newstate, bool propagate)
     // set it to above. OnStateChange can call SetState recursively. We
     // don't want to notify/propagate an old state.
     if (propagate && (state == newstate)) {
+      notify(NFY_VERBOSE, "We wish to propagate and state: %s is equal to newstate: %s", 
```

Comment:
This notify contains a subset of the information that the next notify has, consider removing it